docs/contribution/releases/building-and-publishing.md

```diff
@@ -0,0 +1,48 @@
+---
+post_title: Building and Publishing a Release
+sidebar_label: Building and Publishing
+---
+
+# Building and Publishing a Release
+
+## Prerequisites
+
+* You should have a WordPress.org account with committer access to the WooCommerce plugin, as well as the associated SVN credentials.
```

Comment:
While this is necessary when performing the steps 'manually', with the new workflows it shouldn't be. The repo should already be configured with the relevant secrets.

Reply:
@jorgeatorres, while the release lead doesn't need SVN credentials, they do still need committer access to approve the release to WP.org, correct?